src/components/Banner/Banner.module.css

```diff
@@ -176,16 +199,87 @@
 
   background: var(--eds-theme-color-background-utility-error);
   color: var(--eds-theme-color-text-utility-error);
+
+  a:hover,
+  a:focus {
+    color: var(--eds-color-other-ruby-800); /* 2 */
+  }
 }
 
 /**
  * Banner brand
  * 1) The thick left border matches the icon color
+ * 2) Links inside the Banner get their color from the banner variant styles
  */
 .banner--brand {
   --border-light-color: var(--eds-theme-color-border-brand-primary-subtle);
   --border-dark-color: var(--eds-theme-color-icon-brand-primary); /* 1 */
 
   background: var(--eds-theme-color-background-brand-primary-default);
   color: var(--eds-theme-color-text-brand-primary);
+
+  a:hover,
+  a:focus {
+    color: var(--eds-color-brand-grape-800); /* 2 */
+  }
+}
+
+/*------------------------------------*\
+    # ACTION STYLES
+\*------------------------------------*/
+
+.banner__action > * {
+  /* TODO: move to mixin */
+  @mixin eds-theme-typography-button-label;
+  display: inline-flex;
+  align-items: center;
+  justify-content: center;
+  margin: 0; /* 3 */
+  gap: 0.75rem; /* 4 */
+  border-width: var(--eds-theme-border-width);
+  border-style: solid;
+  border-radius: var(--eds-theme-border-radius);
+  cursor: pointer;
+  transition: all var(--eds-anim-fade-quick) var(--eds-anim-ease);
+
+  &:focus {
+    @mixin focus;
+  }
+
+  &:disabled,
+  &:disabled:hover {
+    cursor: not-allowed;
+    border-color: var(--eds-theme-color-border-disabled);
+    background-color: transparent;
+    color: var(--eds-theme-color-text-disabled);
+  }
+
+  svg {
+    --icon-size-default: 2em; /* 5 */
+  }
+
+  @media screen and (prefers-reduced-motion) {
+    transition: none;
+  }
+}
+
+/* stylelint-disable-next-line no-descending-specificity */
+.banner--success .banner__action > * {
```

Comment:
I only added the success action here as an example of what it would look like (again, I didn't want to dump too much time into this). I used `*` instead of `button` here because it's possible this button is actually going to be a link.